pkg/plugins/runtime/gateway/testdata/http/01-gateway-route.yaml

```diff
@@ -1,6 +1,6 @@
 Clusters:
   Resources:
-    echo-service-5a416c39037aa8f6:
+    echo-service-9f149ed9e14091ca:
```

Comment:
If we are changing the way we do format.

I think we should change it to underscore so it would be
```
echo-service_9f149ed9e14091ca_
```
or other compliment way with metrics hijacker.
@lobkovilya can you chime in?

Reply:
ah, interesting. Ok then. This will work.